bridge/bridge.go

```diff
@@ -97,8 +100,7 @@ func (b *Bridge) Sync(quiet bool) {
 
 	log.Printf("Syncing services on %d containers", len(containers))
 
-	// NOTE: This assumes reregistering will do the right thing, i.e. nothing.
-	// NOTE: This will NOT remove services.
+	// NOTE: This assumes reregistering will do the right thing, i.e. nothing..
```

Comment:
This is the current behavior, this PR does not change that and changing that behavior would be out of scope of this.